### ISSUE TITLE
Custom Typeface?

### ISSUE THREAD

**Issue by truXton222** [blennerSilva/AwesomeDialog#2]
Dialog is very cool, but i can't find a way to customize Typeface.

Is possible? You can add a feature to customize all dialog typeface(title, message,buttons, ecc.)

Anyway, very good work!

**Comment by blennerSilva** [comment]
Yes sure, i will work on a new release very soon with more dialogs and can def work on fonts
